### ISSUE TITLE
Different font size settings for each device type

### ISSUE THREAD

**Issue by jarrah31** [isontheline/pro.webssh.net#588]
# Feature description
Would it be possible to define font sizes per device type (iPhone/iPad/Desktop) in a future WebSSH version please? 

For example, I prefer 12px on my iPad because the device is often closer to my eyes. However, 12px is too small on my MacBook so I'd prefer 15px there instead. 

Thanks.

**Comment by isontheline** [comment]
That's right, font size setting has not been implemented inside macOS binary.

Thanks for this catch!

**Comment by isontheline** [comment]
Hi @jarrah31 👋

Which option is the best?
1. Show three settings inside the connection form for all devices types (iPhone size, iPad size, macOS size)
2. Show only one setting (Size) which will show only the size of the current device (and not the other devices). So other devices sizes will need to be set on their own

**Comment by jarrah31** [comment]
Hi Arnaud,

Thanks for looking into this!

I think option 2 would be best so that font size changes are only applied to the device where the change is made. Perhaps make a comment within the settings to inform users that these are device specific so that no one complains when they notice that font size changes don't sync?

**Comment by isontheline** [comment]
Thank you a lot for your feedback @jarrah31 🙏

So I will work on option 2 in the following days ;)

> Perhaps make a comment within the settings to inform users that these are device specific so that no one complains when they notice that font size changes don't sync?

Yes you're right a comment could be a great addition

**Comment by isontheline** [comment]
Hello @jarrah31 👋

Now available in the latest beta release WebSSH 19.7

Have a nice day ☀️

**Comment by jarrah31** [comment]
It works great, thank you very much!!

While testing this feature, I noticed that if I make changes in a Connection, either within Terminal Settings or something like the username, and I press the back arrow button, I can easily lose any changes I made without realising (the small tick icon isn't very intuitive). Would it be possible to notify the user that changes were detected and maybe have it ask if they want to discard them, reminding the user to press the tick icon to save? Or perhaps have a more obvious "Save" button that turns red if changes are detected? Thanks :)
